src/Form/ResetPasswordRequestFormType.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+class ResetPasswordRequestFormType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('email', EmailType::class, [
+                'attr' => ['autocomplete' => 'email'],
```

Comment:
```suggestion
                'attr' => [
                    'autocomplete' => 'email',
                    'class' => 'mt-3',
                ],
```

si tu veux ajouter de la marge en haut de ton champ en tailwind ça devrait ressembler à ça

Reply:
Ok, parfait, merci du coup de main